cdap-master/src/main/java/co/cask/cdap/data/tools/ReplicationStatusTool.java

```diff
@@ -417,7 +419,9 @@ private static void checkHDFSReplicationComplete(SortedMap<String, String> maste
       System.out.println("Number of HDFS files on the Master and Slave Clusters do not match.");
     }
 
-    complete = !checkDifferences(masterChecksumMap.keySet(), slaveChecksumMap.keySet(), "File");
+    // Verify all files on Master are present on Slave. Ignore any extra files on slave. This could happen
+    // when old snapshot files are pruned by cdap on Master cluster.
```

Comment:
```
    // Verify that all files on Master are present on Slave. Ignore any extra files on Slave. This could 
    // happen when old snapshot files are pruned by CDAP on the Master cluster.